crates/dbs-virtio-devices/src/balloon.rs

```diff
@@ -601,7 +601,7 @@ where
         self.device_info.set_acked_features(page, value)
     }
 
-    fn read_config(&mut self, offset: u64, mut data: &mut [u8]) {
+    fn read_config(&mut self, offset: u64, mut data: &mut [u8]) -> bool {
```

Comment:
Will return `Result<()>` is better here? It's more strightforward than `bool` .

Reply:
Done